frontend/components/utils/Explain.vue

```diff
@@ -46,6 +46,10 @@ export default Vue.extend({
             type: String,
             default: 'topRight',
         },
+        setWidth: {
```

Comment:
Do we want to call this setWidth? Can we just use the term `width` or is there any kind of overlap?